docs/src/main/paradox/project/migration-guide-2.4.x-2.5.x.md

```diff
@@ -1,4 +0,0 @@
-# Migration Guide 2.4.x to 2.5.x
```

Comment:
as mentioned, I removed the detailed migration guides that apply for older Akka versions